server/graphql/v1/CollectiveInterface.js

```diff
@@ -1183,13 +1183,7 @@ const CollectiveFields = () => {
       async resolve(collective, _, req) {
         if (req.remoteUser?.isAdmin(collective.id) || req.remoteUser?.isRoot()) {
           if (collective.type === types.USER) {
-            const user = await models.User.findOne({
-              attributes: ['id', 'twoFactorAuthToken'],
-              where: { CollectiveId: collective.id },
-            });
-            if (user.twoFactorAuthToken) {
-              return true;
-            }
+            return await req.remoteUser.hasTwoFactorAuthentication();
```

Comment:
@hdiniz : The removal of this part introduces a bug. Now note that the 2FA flag for the remote user is returned always. But we need to make sure that the the flag returns whether the 2FA is enabled for the given collective. 

This is important in root actions for example, where with this change it seems that now, even if you disable 2FA for a collective; it still showing as the collective has 2FA enabled (screenshot below; I've disabled for this collective) as the flag is actually coming from my account (where 2FA is enabled). 

![image](https://github.com/opencollective/opencollective-api/assets/12435965/d2e57846-5390-4f4d-91d9-70d28f60be55)